.github/workflows/build-images-ci.yaml

```diff
@@ -2,7 +2,7 @@ name: Image CI Build
 
 # Any change in triggers needs to be reflected in the concurrency group.
 on:
-  pull_request_target:
```

Comment:
Is this itentional?

Reply:
oh this is from a temporary commit to test the CI image build: https://github.com/cilium/cilium/pull/34178/commits/e5d1461539526759f3f52f8111d0062425f5be67 let me remove it 👍